python/test.py

```diff
@@ -0,0 +1,68 @@
+from getopt import GetoptError, getopt
+from re import A
+import sys
+
+
+#Help page // Displays how to use the script
+def usage_collecter_usage():
+    print("""Usage: resource-monitor.py 
+
+    OPTIONS:
+    -p or --path        : Optional, Used to designate path loaction for the .csv file.
+                          resource-monitor.py -p /desired/path/location
+                          resource-monitor.py --path /desired/path/location
+
+    -t or --time        : Optional, Used to designate time interval at which data is collected for the .csv file. (In seconds)
+                          resource-monitor.py -t 10   
+                          resource-monitor.py --time 10                   
+
+    -h or --help        : Optional, Shows all options and how to use them.
+    """)
+
+def get_arguments():
+
+    try:
+        #option map
+        options = getopt(sys.argv[1:],
+                        shortopts="p:t:h",    
+                        longopts=["path=", "time=", "help"])
+    except GetoptError as e:
+        print("ERROR, wrong option used: ", e)
+        sys.exit(usage_collecter_usage())
+   
+    if not options[0]:
```

Comment:
This line (33) is different.   Because of the if statement, when any one or more of the parameters is passed, neither path nor time will get initalized to a default.   So if you called it with the '-p' option, then time becomes undefined at the return statement.